crates/wasm/src/client/implementation/specific.rs

```diff
@@ -187,6 +192,45 @@ impl wit::client_audio::Host for Bindings {
         });
         Ok(())
     }
+
+    fn stop(&mut self, url: String) -> anyhow::Result<()> {
+        let world = self.world();
+        let runtime = world.resource(runtime()).clone();
+        let async_run = world.resource(async_run()).clone();
+        runtime.spawn(async move {
+            async_run.run(move |world| {
+                let sender = world.resource(audio_sender());
+                sender.send(AudioMessage::Stop(url)).unwrap();
+            });
+        });
+        Ok(())
+    }
+
+    fn set_amp(&mut self, url: String, amp: f32) -> anyhow::Result<()> {
```

Comment:
`set_volume` - even if the users don't see it, it'll be confusing for us + anyone else who ends up writing WIT bindings for their language

Reply:
fixed